docs/source/basic/cheatsheet.rst

```diff
@@ -163,6 +164,11 @@ Enqueue a memory copy from host to device
 
      memcpy(queue, bufDevice, bufHost, extent);
 
+Enqueue a memory copy from host to device
+  .. code-block:: c++
+
+     memcpy(queue, bufDevice, hostView, extent);
```

Comment:
`bufDevice` can be also a view. I suggest to place a comment on the original `memcpy` and write

// bufDevice and/or bufHost can be also alpaka::view's